src/ucp/api/ucp.h

```diff
@@ -1607,6 +1696,26 @@ int ucp_request_is_completed(void *request);
 
 /**
  * @ingroup UCP_COMM
+ * @brief Check the status of non-blocking request.
+ *
+ * This routine checks the state of the request and returns its current status.
+ * Any value different from UCS_INPROGRESS means that request is in a completed
+ * state.
+ *
+ * @param [in]  request     Non-blocking request to check.
+ *
+ * @param [out] info        If request is in completed state, it is
+ *                          filled with the details about the message.
+ *
+ * @note The @p info parameter is relevant for receive operations only.
```

Comment:
You have specify what does it mean for send ? Do we leave it uninitialized or set it to NULL ?